tables/sr-cyr.tbl

```diff
@@ -0,0 +1,21 @@
+#-index-name: Serbian-Cyrillic
+#-display-name: Serbian Cyrillic braille
+
+#+language:sr
+#+region:sr-cyr
+#+type:literary
+#+grade:1
```

Comment:
Does anybody ever refer to the "grade" of Serbian braille? If not, this field can be dropped.

Regardless, we should have a `#+contraction` field. I'm not sure whether we can consider the "SPECIAL SYLLABLES" rules contractions or not. I guess the answer is no. But I wasn't sure.